rel/overlay/etc/default.ini

```diff
@@ -14,7 +14,7 @@ view_index_dir = {{view_index_dir}}
 ; The actual limit may be slightly lower depending on how
 ; many schedulers you have as the allowance is divided evenly
 ; among them.
-;max_dbs_open = 500
+max_dbs_open = 500
```

Comment:
This seem like an unrelated change

Reply:
`APPDIR` and this one are used to pass `make eunit`.